### ISSUE TITLE
logging serial console to a file

### ISSUE THREAD

**Issue by brianjmurrell** [vagrant-libvirt/vagrant-libvirt#901]
Does vagrant-libvirt support being able to log the VM console to a file.  Apparently:

```xml
    <serial type='file'>
      <source path='/var/log/libvirt/qemu/serial/bb-2.6.18-128.el5-x86_64.log'/>
      <target port='0'/>
    </serial>
```

will log a VM's serial port to a file.  Is this construct possible with vagrant-libvirt?

Or is there any other method of logging a VM's console to a file?

**Comment by electrofelix** [comment]
Take a look at #695 which contains the base support, but it needs to be updated and synced with the latest codebase. 2 main issues resulted, in that the log file would be owned by root, and you lost access to connect into the serial console so had been meaning to look into a better default that could forward the output to a user process that could two wait serial communication to log to a file for what was outputted while still permitting access for users needing to connect into the console.

**Comment by electrofelix** [comment]
Forgot to mark this as fixed with #1385, it can be improved on to better handle the access issue, but the underlying support has landed.
